src/types.ts

```diff
@@ -12,7 +12,8 @@ export type LambdaHttpMethod =
   | 'OPTIONS'
   | 'PATCH'
   | 'POST'
-  | 'PUT';
+  | 'PUT'
+  | 'ANY';
```

Comment:
I have only recently removed the Catch-ANY functionality. In my memory an ANY route on AWS had not worked. Were you able to test this successfully with a real deployed stack?

See:
https://github.com/clebert/aws-simple/commit/f24ae9a6f722040cff3180bfd6df773425f5b51e

Reply:
You're right it doesn't not work with deployment. I will remove that part.